assets/src/domain/shared/entities/datetimes/predicates/filters/index.ts

```diff
@@ -16,7 +15,7 @@ import soldOutOnly from './soldOutOnly';
 import trashedOnly from './trashedOnly';
 import upcomingOnly from './upcomingOnly';
 
-export const now = new Date();
+export const now = parseISO(formatISO(new Date()));
```

Comment:
nice 👍

Reply:
I've used your approach and lifted up to be able to use that in all other instances.
Thx for that fix.